### PR TITLE
fix issue #1276

### PR DESCRIPTION
**Pull request by lzhan**
resolve functional tests failures on node8 after pr1264 is merged.

### CODE REVIEW

File: lib/output-handler.server.js [hunk 1]
```diff
@@ -123,8 +123,10 @@ OutputHandler.prototype = {
 
 
     _writeHeaders: function() {
-        if (!this.res.headersSent) {
+        //support both node8 and node10
```
Comment (drewfish) [inline lib/output-handler.server.js]: I'd maybe say something addition, such as "res.headersSent was introduced in node10".

**Comment by drewfish** [comment]
+1

**Comment by caridy** [comment]
Let's make sure we add a trello card, or an issue in github to remove the unnecessary code once we drop support for 0.8

**Comment by lzhan** [comment]
@caridy Thanks for the reminder. I am adding a card now.

**Pull request merged**